test/integration/sonobuoy_integration_test.go

```diff
@@ -495,32 +495,36 @@ func TestExactOutput_LocalGolden(t *testing.T) {
 			expectFile: "testdata/gen-variable-image.golden",
 		}, {
 			desc:       "gen doesnt provide UUID",
-			cmdLine:    "gen",
+			cmdLine:    "gen --kubernetes-version=ignore",
 			expectFile: "testdata/gen-no-uuid.golden",
 		}, {
 			desc:       "gen config doesnt provide UUID",
 			cmdLine:    "gen config",
 			expectFile: "testdata/gen-config-no-uuid.golden",
 		}, {
 			desc:       "gen with config testing fields that also have flags",
-			cmdLine:    "gen --config=testdata/subfieldTest.json",
+			cmdLine:    "gen --config=testdata/subfieldTest.json --kubernetes-version=ignore",
```

Comment:
These tests were just failing locally because there was no cluster; I was trying to avoid that. I can remove this if needed, but I hated that when running `update_local` from the bash scripts it had an issue with numerous of these tests because it was previously catching that I had a v1.20.0 k8s cluster locally.